paparazzi/src/main/java/app/cash/paparazzi/internal/apng/ApngVerifier.kt

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package app.cash.paparazzi.internal.apng
+
+import app.cash.paparazzi.internal.ImageUtils
+import app.cash.paparazzi.internal.ImageUtils.resize
+import okio.FileSystem
+import okio.Path
+import org.junit.Assert.fail
+import java.awt.image.BufferedImage
+import java.io.Closeable
+import kotlin.math.max
+import kotlin.math.min
+
+internal class ApngVerifier(
+  goldenFilePath: Path,
+  private val deltaFilePath: Path,
+  private val fps: Int,
+  private val frameCount: Int,
+  private val maxPercentDifference: Double,
+  private val fileSystem: FileSystem = FileSystem.SYSTEM,
+  private val withErrorText: Boolean = true
```

Comment:
it looks like this is only set explicitly to false in tests.

1) is there a potential other use case for disabling?
2) if not, why not remove the field and always print the text?  (guess: hard to get consistent test successes over all platforms?)

Reply:
The latter because the text is jdk rendered we get different default font's and subtle rendering differences even when trying to force the same font on different platforms, would have to fallback to a % diff test instead of exact image match which didn't feel great